content/en/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -331,7 +331,7 @@ Currently, resources are created without this annotation, so the first invocatio
 All subsequent calls to `kubectl apply`, and other commands that modify the configuration, such as `kubectl replace` and `kubectl edit`, will update the annotation, allowing subsequent calls to `kubectl apply` to detect and perform deletions using a three-way diff.
 
 {{< note >}}
-To use apply, always create resource initially with either `kubectl apply` or `kubectl create --save-config`.
+To use apply, always create resource initially with either `kubectl apply`.
 {{< /note >}}
```

Comment:
@Liujingfang1, does this note still make sense [with either]? At the very least, clarify the note.

Reply:
I took a look at the preceding paragrahps, which explains `kubectl apply`. I think we can remove the note.